app/app.py

```diff
@@ -0,0 +1,39 @@
+import asyncio
+import signal
+from datetime import datetime
+from .process_manager import ProcessManager, TaskType
+from .scheduler import Scheduler
+
+TASK_QUEUE = []
+
+
+class App:
+    def __init__(self, config):
```

Comment:
Can we say anything about the type of `config`?

Reply:
This is address in #120